src/main/java/org/lisoft/lsml/model/database/gamedata/Localisation.java

```diff
@@ -44,7 +44,10 @@ public class Localisation {
     public static void initialize(GameVFS aGameVFS) throws IOException {
         key2string = new HashMap<>();
 
-        final File[] files = new File[] { new File("Game/Localized/Languages/TheRealLoc.xml") };
+        final File[] files = new File[] { new File("Game/Localized/Localization/English/TheRealLoc.xml") };        
+        /*
```

Comment:
Please prefer to not leave commented out code (ignore the fact that I forgot to clean up my old commented out code plz lol, I've gotten better at it since I wrote this). If we need to find it, we can find it in the history :)

Reply:
Looks good other than the one comment above.  Happy to merge this.

Btw, github has this feature where if you tag an issue in the commit like you did with #773, it will be linked to from the issue. Also if you put certain keywords like "fixes #773" the issue will be automatically closed when the commit is merged. Or at least it does for me, I hope it does so for pull requests too...